### ISSUE TITLE
AWSAppSync 3.1.6 with AWSCore 2.17 pod install failure

### ISSUE THREAD

**Issue by afriedmanGlacier** [aws-amplify/aws-sdk-ios#3089]
Prior to the release of AWSCore 2.17, we were seeing the issue mentioned in https://github.com/aws-amplify/aws-sdk-ios/issues/2927

We got past this issue by downgrading AWSUserPoolsSignIn (from 2.16.0) to 2.15.1. In our case, that also meant downgrading AWSAppSync to 3.1.5 due to competing dependencies.

Now that 2.17 fixes the build issues with XCode 12, we still cannot use AWSAppSync 3.1.6. If we run a pod install, this is the response:

```
[!] CocoaPods could not find compatible versions for pod "AWSCore":
  In snapshot (Podfile.lock):
    AWSCore (= 2.17.0)

  In Podfile:
    AWSAppSync (~> 3.1.6) was resolved to 3.1.6, which depends on
      AWSCore (~> 2.16.0)

    AWSCognitoIdentityProvider was resolved to 2.17.0, which depends on
      AWSCore (= 2.17.0)

Specs satisfying the `AWSCore (= 2.17.0), AWSCore (= 2.17.0), AWSCore (~> 2.16.0)` dependency were found, but they required a higher minimum deployment target.
```

**Comment by lawmicha** [comment]
Hi @afriedmanGlacier, sorry for the inconvenience, we are currently releasing the next AppSync SDK version that will work with 2.17.0. Wil update once the pod is published.

**Comment by lawmicha** [comment]
AppSync 3.1.8 has been released, https://github.com/awslabs/aws-mobile-appsync-sdk-ios/releases/tag/3.1.8 please let us know if you are till having troubles using AppSync

**Comment by afriedmanGlacier** [comment]
Analyzing dependencies
[!] CocoaPods could not find compatible versions for pod "AWSCore":
  In snapshot (Podfile.lock):
    AWSCore (= 2.17.0)

  In Podfile:
    AWSAppSync (~> 3.1.8) was resolved to 3.1.8, which depends on
      AWSCore (~> 2.16.0)

    AWSCognitoIdentityProvider was resolved to 2.17.0, which depends on
      AWSCore (= 2.17.0)

Still having this issue.

**Comment by palpatim** [comment]
@afriedmanGlacier Sorry, it looks like we didn't correctly update the Podspec for the AWSCore dependency! We'll get that fixed and re-published shortly.

**Comment by lawmicha** [comment]
Hi @afriedmanGlacier, AppSync 3.1.9 has been released with the fix on AWSCore dependency to use 2.17.0. please give this one a try

**Comment by afriedmanGlacier** [comment]
Looks better, thanks!
